pkgs/development/python-modules/nix-prefetch-github/default.nix

```diff
@@ -5,23 +5,31 @@
 , which
 , pythonOlder
 , unittestCheckHook
+, sphinxHook
+, sphinx-argparse
+, parameterized
 }:
 
 buildPythonPackage rec {
   pname = "nix-prefetch-github";
-  version = "7.0.0";
+  version = "7.1.0";
   format = "setuptools";
```

Comment:
While this still works, this was deprecated in favor of `pyproject = true` (see https://github.com/NixOS/nixpkgs/pull/254136).

Yes, you can set `pyproject = true` even if your package itself doesn't use `pyproject` since it will fallback to use `setuptools` (you will need to add `setuptools` to `nativeBuildInputs` though).